docs/specs/single-file.yml

```diff
@@ -29,6 +31,10 @@ inputs:
 outputs:
   a/a.json:
   b/b.json:
+  a/.publish.json: |
```

Comment:
update test cases for `.links.json`? since it does not contain the entire build scope either